docs/ossec_alerts.rst

```diff
@@ -475,6 +475,35 @@ email alert every morning containing text similar to: ::
 
 This is a normal alert, and informs you that the system is working as expected.
 
+Similarly, every few weeks your SecureDrop Application and Monitoring
```

Comment:
The servers are checking for updates every day and then updating if necessary, so remove the "every few weeks" bit